plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/testdata/testpom.xml

```diff
@@ -13,6 +13,18 @@
   <artifactId>com.google.cloud.tools.eclipse.appengine.libraries.test</artifactId>
   <version>0.1.0-SNAPSHOT</version>
   <packaging>eclipse-test-plugin</packaging>
+  
+  <dependencyManagement>
```

Comment:
Is this `dependencyManager` or `dependencyManagement`? (I'm confused.)

Reply:
dependencyManagement